src/libraries/Common/src/Interop/Windows/NCrypt/Interop.ErrorCode.cs

```diff
@@ -20,6 +20,12 @@ internal enum ErrorCode : int
             NTE_NO_MORE_ITEMS = unchecked((int)0x8009002a),
             E_FAIL = unchecked((int)0x80004005),
             STATUS_UNSUCCESSFUL = unchecked((int)0xC0000001),
+            TPMAPI_E_BUFFER_TOO_SMALL = unchecked((int)0x80290406),
```

Comment:
According to https://learn.microsoft.com/en-us/windows/win32/com/com-error-codes-6 the value of `TPMAPI_E_BUFFER_TOO_SMALL` is equal to `0x80290106`. I am guessing it's a typo?

```suggestion
            TPMAPI_E_BUFFER_TOO_SMALL = unchecked((int)0x80290106),
```

Reply:
Ah I used the right value but the wrong name. It should be `TPM_E_PCP_BUFFER_TOO_SMALL`. Thanks for checking!